src/FakeItEasy/ValueProducerSignatureHelper.cs

```diff
@@ -22,6 +24,20 @@ internal static void AssertThatValueProducerSignatureSatisfiesCallSignature(Meth
             }
         }
 
+        internal static void AssertThatValuesSatisfyCallSignature(MethodInfo callMethod, object[] values)
+        {
+            if (IsCallSignatureSatisfiedByValues(callMethod, values))
+            {
+                return;
+            }
+
+            var fakeSignature = BuildSignatureDescription(callMethod.GetParameters().Select(p => p.ParameterType));
+            var actionSignature = BuildSignatureDescription(values.Select(v => v?.GetType()));
+
+            throw new FakeConfigurationException(
+                $"The event has the signature ({fakeSignature}), but the provided arguments have types ({actionSignature}).");

```

Comment:
Well, in this case I think we should include `()`, because  it's a list of parameters. Anyway, we can revisit if we decide to go ahead with this idea.